src/util.ts

```diff
@@ -368,6 +368,12 @@ export function getToolsEnvVars(): any {
 	return envVars;
 }
 
+export function substituteEnv(input: string): string {
+	return input.replace(/\${env:([^}]+)}/mg, function (match, capture) {
```

Comment:
I believe we dont need the `m` flag here. Any reason we should have it?

Reply:
I can't assume input string is single line. someone could use that function elsewhere.

Reply:
Thats true, but isnt the `m` flag make sense only if you are using either `^` or `$` in the regular expression?

Reply:
sorry for late response, I'll remove `m` flag and add some tests within few hours